pkg/network/ebpf/c/port_range.h

```diff
@@ -3,13 +3,20 @@
 
 #include "ip.h"
 
-// TODO: Replace those by injected constants based on system configuration
-// once we have port range detection merged into the codebase.
-#define EPHEMERAL_RANGE_BEG 32768
-#define EPHEMERAL_RANGE_END 60999
+static __always_inline __u16 ephemeral_range_begin() {
+    __u64 val = 0;
+    LOAD_CONSTANT("ephemeral_range_begin", val);
+    return (__u16) val;
+}
+
+static __always_inline __u16 ephemeral_range_end() {
+    __u64 val = 0;
+    LOAD_CONSTANT("ephemeral_range_end", val);
+    return (__u16) val;
+}
 
 static __always_inline int is_ephemeral_port(u16 port) {
-    return port >= EPHEMERAL_RANGE_BEG && port <= EPHEMERAL_RANGE_END;
+    return port >= ephemeral_range_begin() && port <= ephemeral_range_end();
```

Comment:
Can you explain why we're making this change in this PR ?  It looks like this is only called in USM related code paths

Reply:
Initially I thought I could do the ephemeral port check eBPF, but that is not feasible, so this is a holdover from that. This needs to be fixed though.